docs/orchestrator.md

```diff
@@ -118,9 +120,23 @@ Starts the Blobstream orchestrator to sign attestations
 
 Usage:
   blobstream orchestrator start <flags> [flags]
+
+Flags:
+      --core.grpc string           Specify the celestia app grpc address (default "localhost:9090")
+      --core.rpc string            Specify the celestia app rest rpc address (default "tcp://localhost:26657")
+      --evm.account string         Specify the EVM account address to use for signing (Note: the private key should be in the keystore)
+      --evm.passphrase string      the evm account passphrase (if not specified as a flag, it will be asked interactively)
+      --grpc.insecure              allow gRPC over insecure channels, if not TLS the server must use TLS
+  -h, --help                       help for start
+      --home string                The Blobstream orchestrator home directory (default "/Users/joshstein/.orchestrator")
+      --log.format string          The logging format (json|plain) (default "plain")
+      --log.level string           The logging level (trace|debug|info|warn|error|fatal|panic) (default "info")
+      --p2p.bootstrappers string   Comma-separated multiaddresses of p2p peers to connect to
+      --p2p.listen-addr string     MultiAddr for the p2p peer to listen on (default "/ip4/0.0.0.0/tcp/30000")
+      --p2p.nickname string        Nickname of the p2p private key to use (if not provided, an existing one from the p2p store or a newly generated one will be used)
 ```
 
-Also, you can set the necessary configuration in the orchestrator's TOML config file. You can find the latter in the orchestrator's home directory under `config/config.toml`.
+Also, you can set the necessary configuration in the orchestrator's TOML config file. You can find the orchestrator's TOML config file in the orchestrator's home directory under `config/config.toml`.
```

Comment:
```suggestion
Also, you can set the necessary configuration in the orchestrator's TOML config file. You can find the orchestrator's TOML config file in the orchestrator's home directory under `config/config.toml`. This would save you the pain of setting all the flags in the command.
```